flowzone.yml

```
@@ -1881,6 +1881,10 @@ jobs:
 
       - name: Generate stacks matrix
         id: cloudformation_stacks
+        env:
+          MERGE_SHA: ${{ needs.versioned_source.outputs.sha }}
+          DEFAULT_BRANCH: ${{ github.base_ref }}
```

nit, but this is not necessarily the default branch so we should avoid naming it so
```suggestion
          BASE_BRANCH: ${{ github.base_ref }}
```

---

Cheers, I am still trying to find the two commits I need to diff for both sync and merge contexts, I think I have it now..